### ISSUE TITLE
The download instructions aren't up to date (v6.14 instead of v6.16)

### ISSUE THREAD

**Issue by sanmai-NL** [pnpm/pnpm#3819]
Changing the URL to the current version results in an HTTP 404 response.

![Screen Shot 2021-10-02 at 18 36 42](https://user-images.githubusercontent.com/3374183/135725196-9ac1a1ef-5dfd-4c10-a200-c82351efbe8d.png)

**Comment by zkochan** [comment]
We don't upload all the pnpm version to that URL. That pnpm is just used to install the latest pnpm version from the registry, so it doesn't necessarily need to be the latest

**Comment by sanmai-NL** [comment]
From which registry, do you mean? 🙂 The NPM Registry does have the latest version.

**Comment by zkochan** [comment]
Yes, the registry has the latest version but `get.pnpm.io` is from here: https://github.com/pnpm/get.pnpm.io

It has some pnpm version uploaded but not all. We only use those to install the latest pnpm from the npm registry.

**Comment by sanmai-NL** [comment]
@zkochan Can you explain this on your download page?

Why don't you offer a version with security patches as latest version?

What's the relation between versioning (SemVer) and your release policy?

**Comment by zkochan** [comment]
If you need the latest, you can use unpkg to do

**Comment by zkochan** [comment]
This targets a part of pnpm that no longer exists, so closing.

The pinned installer URL in that screenshot is gone. https://pnpm.io/installation documents the current standalone script, the OS package managers and the `packageManager` field.

If you still hit this on the latest pnpm, please comment with a reproduction and we will reopen.

---
Written by an agent (Claude Code, claude-opus-5).
